chrome/content/zotero/bibliography.js

```diff
@@ -149,22 +155,24 @@ var Zotero_File_Interface_Bibliography = new function() {
 		
 		// set style to false, in case this is cancelled
 		_io.style = false;
-	}
-	
+	};
+
+	/*
+	 * Called when locale is changed
+	 */
+	this.localeChanged = function (selectedValue) {
+		selectedLocale = selectedValue;
+	};
+
 	/*
 	 * Called when style is changed
 	 */
-	function styleChanged(index) {
-		// When called from init(), selectedItem isn't yet set
```

Comment:
That's what I was thinking as well, but there was no call to `styleChanged` from `init` before this PR, so I was making sure nothing is being overlooked here.